### PR TITLE
Docs: update CHANGELOG on v0.9.28

### PR DESCRIPTION
**Pull request by kikuomax**
## Proposed Changes

- Adds changes in v0.9.28 since `CHANGELOG` was last udpated

### CODE REVIEW

**Review by wesdevpro — APPROVED** [review]

**Comment by kikuomax** [comment]
@wesdevpro Please re-review after merging #3991 and updating `CHANGELOG`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
 
 ## 0.9.28
 
+### New features
+
+* [#3862](https://github.com/buefy/buefy/issues/3862) Expose `isActiveDetailRow` method in the default scoped slot props of table column (thanks @OrestHk)
+* [#3863](https://github.com/buefy/buefy/issues/3863) Add `active` event emitted when dropdown turns active/inactive (thanks @daniil4udo)
+
 ### Fixes
```
Comment (wesdevpro) [inline CHANGELOG.md:11]: @kikuomax are we waiting for results for #3992 to add #3991 to the list of fixes?

Comment (kikuomax) [reply CHANGELOG.md:11]: @wesdevpro No, I do not expect we can conclude #3992 before releasing 0.9.28.

Comment (wesdevpro) [reply CHANGELOG.md:11]: @kikuomax Makes sense to me 👍🏼Just wanted to confirm with you

**Review by wesdevpro — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,20 @@
...
 
 * Fix [#3961](https://github.com/buefy/buefy/issues/3961) Dropdown with "hover" trigger incorrectly rendered on mobile devices
-* Fix [#3963](https://github.com/buefy/buefy/issues/3963) no dropdown showed up if the data array was empty and only the footer slot was registered (thanks @GMartigny)
+* Fix [#3963](https://github.com/buefy/buefy/issues/3963) No dropdown showed up if the data array was empty and only the footer slot was registered (thanks @GMartigny)
+* Fix [#3971](https://github.com/buefy/buefy/issues/3971) Problems with table headers and dates due to 0.9.25 → 0.9.27 changes
+* Fix [#3971](https://github.com/buefy/buefy/issues/3971) Percentage width on table columns doubly applied
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential call after unmount in `NumberInput` (thanks @Arooba-git)
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential call after unmount in `Sidebar` (thanks @Arooba-git)
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential listener leaks in `SliderThumb` (thanks @Arooba-git)
```
Comment (kikuomax) [inline CHANGELOG.md:18]: @wesdevpro Sorry for confusion, but I appended fixes made in #3991 to CHANGELOG.

Comment (wesdevpro) [reply CHANGELOG.md:18]: Sounds good!

**Review by wesdevpro — APPROVED** [review]
Looks great! Thanks for adding #3991.

**Pull request merged**